src/webrtc/callEventHandler.ts

```diff
@@ -87,17 +89,19 @@ export class CallEventHandler {
 
     private onEvent = (event: MatrixEvent) => {
         // any call events or ones that might be once they're decrypted
+        const isBeingDecrypted = event.isBeingDecrypted();
+        const shouldAttemptDecryption = event.shouldAttemptDecryption();
         if (
             event.getType().indexOf("m.call.") === 0 ||
             event.getType().indexOf("org.matrix.call.") === 0
-            || event.isBeingDecrypted()
+            || isBeingDecrypted || shouldAttemptDecryption
         ) {
             // queue up for processing once all events from this sync have been
             // processed (see above).
             this.callEventBuffer.push(event);
         }
 
-        if (event.isBeingDecrypted() || event.isDecryptionFailure()) {
+        if (event.isDecryptionFailure() || isBeingDecrypted || shouldAttemptDecryption) {
```

Comment:
So for call events, we're relying on `decryptCriticalEvents` to attempt the decryption, correct? (I think it seems right, I just wanted to confirm my understanding.)